tests/data/simple_cases/function_trailing_comma.py

```diff
@@ -68,6 +92,34 @@ def f(
     d = {
         "key": "value",
     }
+    e = {
+        "a": fun(msg, "ts"),
+        "longggggggggggggggid": ...,
+        "longgggggggggggggggggggkey": ...,
+        "created": ...,
+        # "longkey": ...
+    }
+    f = [
+        arg1,
+        arg2,
+        arg3,
+        arg4,
```

Comment:
This seems short enough to fit in one line, so it's a little weird to me that we don't. However, current behavior is already to put everything on its own line and that seems fine, so let's leave it as is.